### ISSUE TITLE
Sharing the source code(is the project dead?)

### ISSUE THREAD

**Issue by agirnob** [openphotogrammetry/colmap-cl#8]
I think this project dead and like @opalize has said before in issue #5 if the source open people can chip in and make it avaible to linux all somekind of other things that may help other project

**Comment by Nosenzor** [comment]
Yes that's the same issue for Meshroom-CL. If we had the source code we (community) could continue to develop it!
